plugins/utils/fzf-lua.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib; let
+  cfg = config.plugins.fzf-lua;
+in {
+  options.plugins.fzf-lua =
+    helpers.extraOptionsOptions
+    // {
+      enable = mkEnableOption "fzf-lua";
+
+      package = helpers.mkPackageOption "fzf-lua" pkgs.vimPlugins.fzf-lua;
+
+      profile = mkOption {
+        type = with types; oneOf [(enum ["default" "fzf-native" "fzf-tmux" "fzf-vim" "max-perf" "telescope" "skim"])];
```

```suggestion
        type = types.enum ["default" "fzf-native" "fzf-tmux" "fzf-vim" "max-perf" "telescope" "skim"];
```
You don't need `oneOf` here.